packages/web/components/complex/add-conc-liquidity.tsx

```diff
@@ -710,31 +706,44 @@ const PresetStrategyCard: FunctionComponent<
 
 const PriceInputBox: FunctionComponent<{
   label: string;
-  currentValue: string;
-  onChange: (val: string) => void;
-  onBlur: (e: any) => void;
-  infinity?: boolean;
-}> = ({ label, currentValue, onChange, onBlur, infinity }) => (
-  <div className="flex w-full max-w-[9.75rem] flex-col items-end rounded-xl bg-osmoverse-800 px-2">
-    <span className="caption px-2 pt-2 text-osmoverse-400">{label}</span>
-    {infinity ? (
-      <div className="flex h-[41px] items-center px-2">
-        <Image
-          alt="infinity"
-          src="/icons/infinity.svg"
-          width={16}
-          height={16}
+  forPriceIndex: 0 | 1;
+  addConcLiquidityConfig: ObservableAddConcentratedLiquidityConfig;
+}> = observer(({ label, forPriceIndex, addConcLiquidityConfig: config }) => {
```

Comment:
I think it's preferable to keep the variable name as 'addConcLiquidityConfig'. While it's lengthier, it has the advantage of being self-explanatory, eliminating the need to read through the parameters.

```suggestion
}> = observer(({ label, forPriceIndex, addConcLiquidityConfig }) => {
```